packages/react-badge/src/components/Badge/useBadgeStyles.ts

```diff
@@ -105,6 +245,34 @@ export const useBadgeStyles = (state: BadgeState): BadgeState => {
     state.appearance === 'ghost' && styles.rootGhost,
     state.appearance === 'outline' && styles.rootOutline,
     state.appearance === 'tint' && styles.rootTint,
+    state.appearance === 'filled' && state.color === 'danger' && styles.rootFilledDanger,
+    state.appearance === 'outline' && state.color === 'danger' && styles.rootOutlineDanger,
```

Comment:
do these comparisons once and save them to a variable, the minifier will have an easier time minifying single variables to letters at that point

Reply:
Done